internal/cmds/helper/optimize.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright (c) 2023 OceanBase
+ob-operator is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+
+	http://license.coscl.org.cn/MulanPSL2
+
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+*/
+package helper
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	cmdconst "github.com/oceanbase/ob-operator/internal/const/cmd"
+	"github.com/oceanbase/ob-operator/pkg/helper/model"
+)
+
+const (
+	OptimizedParameterConfigFile = "/home/admin/oceanbase/etc/default_parameter.json"
```

Comment:
How would we put the two files on desired path?

Reply:
These two files are provided by OceanBase package

Reply:
It's already in the image